src/plugins/share/public/components/url_panel_content.tsx

```diff
@@ -186,13 +203,25 @@ export class UrlPanelContent extends Component<Props, State> {
     return this.props.shareableUrl || window.location.href;
   };
 
-  private makeUrlEmbeddable = (url: string) => {
-    const embedQueryParam = '?embed=true';
+  private getEmbedQueryParams = (): string => {
+    return [
+      ['&show-top-nav-menu=true', this.state.showTopNavMenu],
+      ['&show-query-input=true', this.state.showQueryInput],
+      ['&show-date-picker=true', this.state.showDatePicker],
+      ['&hide-filter-bar=true', !this.state.showFilterBar], // Inverted to keep default behaviour for old links
```

Comment:
Can we name all parameters `hide-something` for consistency?

Reply:
I'd prefer show, just so we're consistent with our own flags.

Reply:
If I change it to use `&show-filter-bar=true`, then all previously generated links (which users may have already embedded into websites etc.) will begin hiding the filter bar because this value won't be set.

Reply:
Well, then lets go for `hide-something` like @majagrubic suggested?